doc/source/param/method_star_maker.incl

```diff
@@ -0,0 +1,191 @@
+:p: `Method:star_maker` parameters.
+
+----
+
+:Parameter:  :p:`Method` : :p:`star_maker` : :p:`flavor`
+:Summary: :s:`Which star_maker method to use`
+:Type:   :t:`string`
+:Default: :d:`STARSS`
+:Scope:     :z:`Enzo`
+
+:e:`Options: "STARSS", "stochastic"`
+
+----
+
+:Parameter:  :p:`Method` : :p:`star_maker` : :p:`use_density_threshold`
+:Summary: :s:`Use number density threshold for star formation`
+:Type:   :t:`logical`
+:Default: :d:`false`
+:Scope:     :z:`Enzo`
+
+:e:`Flag to enable number density threshold for star formation.`
+
+----
+
+:Parameter:  :p:`Method` : :p:`star_maker` : :p:`number_density_threshold`
+:Summary: :s:`Use number density threshold for star formation`
+:Type:   :t:`logical`
+:Default: :d:`false`
+:Scope:     :z:`Enzo`
+
+:e:`Set number density threshold for star formation. Requires "use_density_threshold"=true.`
+
+----
+
+Parameter:  :p:`Method` : :p:`star_maker` : :p:`use_overdensity_threshold`
+:Summary: :s:`Use overdensity threshold`
+:Type:   :t:`logical`
+:Default: :d:`false`
+:Scope:     :z:`Enzo`
+
+:e:`Flag to enable overdensity threshold for star formation.`
+
+----
+
+:Parameter:  :p:`Method` : :p:`star_maker` : :p:`overdensity_threshold`
+:Summary: :s:`Use overdensity threshold for star formation`
+:Type:   :t:`float`
+:Default: :d:`0.0`
+:Scope:     :z:`Enzo`
+
+:e:`Set overdensity threshold for star formation. Requires "use_overdensity_threshold"=true.`
+
+----
+
+Parameter:  :p:`Method` : :p:`star_maker` : :p:`use_velocity_divergence`
+:Summary: :s:`Use converging flow criterion for star formation`
+:Type:   :t:`logical`
+:Default: :d:`false`
+:Scope:     :z:`Enzo`
+
+:e:`Flag to check whether div(V) < 0`
+
+----
```

Comment:
On reflection, I would say that I would leave it up to you as to whether you decided to go with this. This could be left for future work. I'll click "resolve conversation" now.